lms/services/digest.py

```diff
@@ -75,8 +81,14 @@ def send_instructor_email_digests(
                 # We don't have an email address for this user.
                 continue
 
+            if deduplicate:
+                task_done_key = f"instructor_email_digest::{unified_user.h_userid}::{datetime.now(timezone.utc).strftime('%Y-%m-%d')}"
+            else:
+                task_done_key = None
```

Comment:
The admin pages pass `deduplicate=False` because we actually do want you to be able to send the same email twice in one day using the admin pages (it'd be a huge pain otherwise)